test/server/fake-backend.js

```diff
@@ -36,7 +36,7 @@ module.exports = (app) => {
     if (req.query['timing-allow-origin'] === 'true') {
       res.set('Timing-Allow-Origin', '*')
     }
-    res.send('ok')
+    setTimeout(() => res.send('ok'), 10)
```

Comment:
What was the issue here?

Reply:
tests too ⚡
duration was at 0 when we expected it to be greater than 0, so I introduced a small delay to ensure that the duration is really > 0

Reply:
Ooh nice :)